.github/workflows/ci.yml

```diff
@@ -236,9 +236,12 @@ jobs:
           path: packages/python/coverage.xml
 
   build-cpp-packages:
-    name: Build C++ Packages
+    strategy:
+      fail-fast: false
+      matrix:
+        type: [tar.gz, deb, rpm]
```

Comment:
I think that a matrix here is not need. You compile 3 times the same thing. Once should be enough

Reply:
No they are 3 different dockerfiles with different underlying architectures. tar.gz uses ubuntu 23.04, deb uses debian 12 and rpm uses rhel7